### ISSUE TITLE
How do you get the ContentViewVersion for the 'Latest' version for an unpublished content view so you can add it to a CompositeContentView

### ISSUE THREAD

**Issue by itewk** [SatelliteQE/nailgun#534]
In the Satellite UI you can add an unpublished ContentView to a CompositeContentView and it will set the version by default to 'Latest'. I am struggling to figure out how you do the same with nailgun. If I try to search for ContentViewVersion with version of 'Latest' that does not work. Some help would be greatly appreciated.

https://github.com/SatelliteQE/nailgun/blob/2d8dfba8d48954cf75ee59fa13a058ac71f0bf68/nailgun/entities.py#L2097

**Comment by itewk** [comment]
You can see my attempts here: https://github.com/theforeman/foreman-ansible-modules/pull/177/files

**Comment by itewk** [comment]
it looks like if you add this to ContentView then you can start to make some progress

```
    def add_content_view_components(self, synchronous=True, components=None, **kwargs):
        kwargs = kwargs.copy()  # shadow the passed-in kwargs
        kwargs.update(self._server_config.get_client_kwargs())
        data = {}
        if components:
          data = { 'components': components }
        response = client.put(self.path('content_view_components/add'), data, **kwargs)
        return _handle_response(response, self._server_config, synchronous)
```

**Comment by sean797** [comment]
Hey @itewk I created https://github.com/SatelliteQE/nailgun/pull/541,

It would be super awesome if you can test it for me :+1:

**Comment by sean797** [comment]
Fixed by #558
